tensorflow/core/util/use_cudnn.h

```diff
@@ -39,6 +39,12 @@ FP16ConvMode CudnnConvComputeMode();
 bool DebugCudnnRnn();
 bool DebugCudnnRnnUseTensorOps();
 int64 DebugCudnnRnnAlgo();
+
+// Return true if the CuDNN depthwise convolution can be used. See cudnn release
```

Comment:
Should be “Returns true” as per google style guide.

Reply:
Done. PTAL.